docs/json/radarr/cf/my5.json

```
@@ -0,0 +1,34 @@
+{
+  "trash_id": "eacd2cb6c6d4a93e459df2cec6d7b97a",
+  "name": "MY5",
+  "includeCustomFormatWhenRenaming": true,
+  "specifications": [
+    {
+      "name": "MY5",
+      "implementation": "ReleaseTitleSpecification",
+      "negate": false,
+      "required": true,
+      "fields": {
+        "value": "\\b(my5)\\b[ ._-]web[ ._-]?(dl|rip)?\\b"
```

```suggestion
        "value": "\\b(MY5)\\b"
```
I think this should be enough, I highly doubt that this combination of letters and numbers is a common movie/show/episode title 😄